Client/Application/AppDelegate.swift

```diff
@@ -189,10 +190,88 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         // Run an invalidate when we come back into the app.
         profile.panelDataObservers.activityStream.invalidate(highlights: true)
 
+        setUpDeepLinks(application: application)
+
         log.debug("Done with setting up the application.")
         return true
     }
 
+    func setUpDeepLinks(application: UIApplication) {
```

Comment:
Nit: Could we move this code to somewhere else? Maybe the `LeanplumIntegration`? Its just that we already have quite a bit of code in the app delegate and whatever we can do to keep it down would help.